cypress/integration/fee-fine/lostItemFee.spec.js

```diff
@@ -0,0 +1,42 @@
+import LostItemFeePolicy from '../../support/fragments/circulation/lost-item-fee-policy';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+
+describe('Fee/fine management', () => {
+  const name = 'A_very_specific_name';
+  const editedName = 'An_edited_name';
+  const duration = '6 week(s)';
+  const lostItemChargeFeeFineError = 'Required if there is a possibility of no fee/fine being charged for a lost item';
+  const duplicateError = 'The Lost item fee policy name entered already exists. Please enter a different name.';
+
+  before('Preconditions', () => {
+    cy.loginAsAdmin({ path: SettingsMenu.circulationLostItemFeePolicyPath, waiter: LostItemFeePolicy.waitLoading });
+  });
+
+  it('C5558 Verify that you can create/edit/delete lost item fee policies', () => {
+    // creating
+    LostItemFeePolicy.startAdding();
+    LostItemFeePolicy.fillName(name);
+    LostItemFeePolicy.save();
+    LostItemFeePolicy.checkErrorMessage(lostItemChargeFeeFineError);
+    LostItemFeePolicy.fillDuration('6', 'week(s)');
+    LostItemFeePolicy.save();
+    LostItemFeePolicy.checkAfterSaving(name, duration);
+
+    // editing
+    LostItemFeePolicy.startEditing();
+    LostItemFeePolicy.fillName(editedName);
+    LostItemFeePolicy.save();
+    LostItemFeePolicy.checkAfterSaving(editedName, duration);
+
+    // creating policy with existing name
+    LostItemFeePolicy.startAdding();
+    LostItemFeePolicy.fillName(editedName);
+    LostItemFeePolicy.fillDuration('6', 'week(s)');
+    LostItemFeePolicy.save();
+    LostItemFeePolicy.checkErrorMessage(duplicateError);
+  });
+  after('Deleting lost item fee policy', () => {
+    // deleting
+    LostItemFeePolicy.delete(name);
```

Comment:
I believe it is better to use API methods to delete the objects that are created before the test. It speeds up the test execution and decrease the number of objects that are not deleted because of any failure in the UI in the test body. I mean any additional possibilities to get the failure during the login, logout, visiting the page needed to delete the item, etc. 
Please use API methods in before, after, beforeAll, afterAll.

Reply:
sure, but it is a part of test case) it checks deleting as well as creating